frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -59,13 +62,18 @@ export const Dashboard = () => {
   const state = history.location.state as {
     sortedOrder: SortOrder
     filterOption: FilterOption
+    tableView: boolean
   }
   const [sortedOrder, setSortedOrder] = useState<SortOrder>(() =>
     state?.sortedOrder ? state.sortedOrder : defaultSortingOrder
   )
   const [filteredOption, setFilteredOption] = useState<FilterOption>(() =>
     state?.filterOption ? state.filterOption : defaultFilterOption
   )
+  const [tableView, setTableView] = useState(() =>
+    state?.tableView === undefined || state?.tableView ? true : defaultView
```

Comment:
change this line to `state?.tableView === undefined ? defaultView : state?.tableView`